app/customer.py

```diff
@@ -0,0 +1,48 @@
+from math import sqrt
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: int,
+            car: dict) -> None:
+
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    def calculate_trip_cost(self, shop: dict, fuel_price: float) -> float:

```

Comment:
The `shop` parameter should be an instance of the `Shop` class, not a dictionary. This ensures that you can access attributes like `location` and `products` directly from the `Shop` instance.